index.html

```diff
@@ -5302,18 +5243,16 @@ <h2 id="canmakepayment-protections">
           <code>canMakePayment()</code> protections
         </h2>
         <p>
-          The {{PaymentRequest/canMakePayment()}} and
-          {{PaymentRequest/hasEnrolledInstrument()}} methods have the potential
-          to expose user information that could be abused for fingerprinting
+          The {{PaymentRequest/canMakePayment()}} method has the potential to
+          expose user information that could be abused for fingerprinting
           purposes. User agents are expected to protect the user from abuse of
           the method. For example, user agents can reduce user fingerprinting
           by:
         </p>
         <ul>
           <li>Allowing the user to configure the user agent to turn off
-          {{PaymentRequest/canMakePayment()}} and
-          {{PaymentRequest/hasEnrolledInstrument()}}, which would return <a>a
-          promise rejected with</a> a {{"NotAllowedError"}} {{DOMException}}.
+          {{PaymentRequest/canMakePayment()}}, which would return <a>a promise
+          rejected with</a> a {{"NotAllowedError"}} {{DOMException}}.
           </li>
           <li>Rate-limiting the frequency of calls with different parameters.
           </li>
```

Comment:
This was originally added for the more fine-grained `hasEnrolledInstrument()`. I think it's not necessary for `canMakePayment()`. What do you think about removing it to keep things simple?

Reply:
Agree...